auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/GrpcUmsClient.java

```diff
@@ -453,6 +454,18 @@ public void deleteMachineUserAccessKeys(String actorCrn, String machineUserCrn,
         }
     }
 
+    /**
+     * Gather anonymization rules for a specific account
+     * NOTE: not supported yet on UMS side
+     * @param accountId account that owns the anonymization rules
+     * @param actorCrn actor that requests to gather the anonymization rules
+     * @return a list of anonymization rules for an UMS account
+     */
+    public List<AnonymizationRule> getAnonymizationRules(String accountId, String actorCrn) {
+        LOGGER.debug("Try getting anonymization rules for {} by {}", accountId, actorCrn);
+        return null;
```

Comment:
The method should be returned with an empty list, we should avoid the unnecessary NPE

Reply:
if you read the description, it's returning null "by design" as it does nothing, that's always the case, it cannot cause NPE .
I can do that you asked but does not make that much sense to change it or not